pepper-apis/studybuilder-cli/studies/osteo/patches/somatic-consent-addendum-ped-v3.conf

```diff
@@ -0,0 +1,71 @@
+{
+  "study": {
+    "guid": "cmi-osteo"
+  },
+  "versionTag": "v3",
+  "activityCode": "CONSENT_ADDENDUM_PEDIATRIC",
+  "block-updates": [
+    {
+      "blockNew":
+        {
+          "titleTemplate": {
+            "templateType": "HTML",
+            "templateText": "$osteo_consent_somatic_contact_pediatric"
+            "variables": [
+              {
+                "name": "osteo_consent_somatic_contact_pediatric",
+                "translations": [
+                  {
+                    "language": "en",
+                    "text": " Who do I contact if I have questions about the research study?"
+                  }
+                ]
+              }
+            ]
+          },
+          "bodyTemplate": {
+            "templateType": "HTML",
+            "templateText": """
+                        <p>$osteo_consent_somatic_contact_p_pediatric</p>
+                        <ul>
+                           <li>$osteo_consent_somatic_contact_li2_pediatric</li>
+                        </ul>
+                        <p>$osteo_consent_somatic_contact_p2_pediatric</p>
+                        """,
+            "variables": [
+              {
+                "name": "osteo_consent_somatic_contact_p_pediatric",
+                "translations": [
+                  {
+                    "language": "en",
+                    "text": """If you have questions about the study, please contact the research doctor or study staff listed below by emailing  <a href="mailto:info@osproject.org" class="Link">info@osproject.org</a> or calling <a href="tel:651-602-2020" class="Link">651-602-2020</a>:"""
+                  }
+                ]
+              },
+              {
+                "name": "osteo_consent_somatic_contact_li2_pediatric",
```

Comment:
Is li2 the right variable name?  If I understand correctly, there's only one entry in the list now.